lib/Factory/ReportScheduleFactory.php

```diff
@@ -170,6 +171,12 @@ public function query($sortOrder = null, $filterBy = [])
             $params['reportName'] = $this->getSanitizer()->getString('reportName',  $filterBy);
         }
 
+        // isActive
+        if ($this->getSanitizer()->getInt('isActive', 0, $filterBy) != 0) {
```

Comment:
I think this should be:

```
$this->getSanitizer()->getInt('isActive', $filterBy) != null
```

so that you can provide 1 or 0 to get active or not, and omit to provide anything to get both?

Reply:
Thanks!